docs/source/getting-started.md

```diff
@@ -96,32 +96,38 @@ on the config system Jupyter uses.
 
 ## Networking
 
-In most situations you will want to change the main IP address and port of the Proxy.
-This address determines where JupyterHub is available to your users.
-The default is all network interfaces (`''`) on port 8000.
+### Configuring the Proxy's IP address and port
+Most use cases will require a change of the Proxy's main IP address and port.
```

Comment:
While we are here, can we change this part? I don't think it's true that in most cases this will want to be changed. I imagine that text may be leftover from the default being localhost in early prototyping. Perhaps just strike the first sentence and leave the rest as-is?